### PR TITLE
add doc of config tidb_max_tiflash_threads for TiFlash

### PR DESCRIPTION
**Pull request by mengxin9014**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a new config tidb_max_tiflash_threads

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Comment by shichun-0415** [comment]
@mengxin9014 Please sign [CLA](https://cla-assistant.io/pingcap/docs-cn). Thanks.
image

File: dynamic-config.md [hunk 1]
```diff
@@ -316,3 +316,42 @@ select @@tidb_slow_log_threshold;
...
+| 10                         |
++----------------------------+
+1 row in set (0.00 sec)
+```
+
+支持在线修改的配置项和相应的 TiDB 系统变量如下：
+
+| 配置项 | 对应变量 | 简介 |
+| --- | --- | --- |
+| max_threads | tidb_max_tiflash_threads | TiFlash 中 request 执行的最大线程数 |
```
Comment (windtalker) [inline dynamic-config.md:357]: 简介改成 “TiFlash 中 request 执行的最大并发度” 吧

Comment (mengxin9014) [reply dynamic-config.md]: > 简介改成 “TiFlash 中 request 执行的最大并发度” 吧

done.

**Review by windtalker — APPROVED** [review]
LGTM

**Comment by shichun-0415** [comment]
@easonn7 是否应该移除文档最开头的实验性功能警告？https://github.com/pingcap/docs-cn/pull/9504/files#diff-1e48553b81f3aea48460b182a6bce36c18c5af15db28286cf4fd9b3a60e078c9R11

File: dynamic-config.md [hunk 2]
```diff
@@ -316,3 +312,42 @@ select @@tidb_slow_log_threshold;
 | log.enable-slow-log | tidb_enable_slow_log | 慢日志的开关 |
 | log.slow-threshold | tidb_slow_log_threshold | 慢日志阈值 |
 | log.expensive-threshold | tidb_expensive_query_time_threshold | expensive 查询阈值 |
+
+### 在线修改 TiFlash 配置
+
```
Comment (qiancai) [inline dynamic-config.md:317]: 目前只有 max_threads 支持在线修改吗？这个可以在这里说下吗？而不是放到最后

File: system-variables.md [hunk 3]
```diff
@@ -1628,6 +1628,22 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
 set tidb_slow_log_threshold = 200;
 ```
 
+### `tidb_max_tiflash_threads`
+
+- 作用域：SESSION | GLOBAL
+- 是否持久化到集群：是
+- 默认值：`-1`
+- 范围：`[-1, 256]`
+- TiFlash 中 request 执行的最大并发度。默认值为 -1，表示该系统变量无效。
```
Comment (qiancai) [inline system-variables.md:1637]: 0 和 1 表示什么呢？

File: system-variables.md [hunk 4]
```diff
@@ -1628,6 +1628,22 @@ Query OK, 0 rows affected, 1 warning (0.00 sec)
...
+
+示例：
+
+{{< copyable "sql" >}}
+
+```sql
+set tidb_max_tiflash_threads = 10;
+```
+
```
Comment (qiancai) [inline system-variables.md:1646]:
```suggestion
```

Comment (qiancai) [reply system-variables.md:1646]: 无需提供例子，系统变量文档开头已经提供了设置变量的语句

File: system-variables.md (hunk 3 above)
Comment (mengxin9014) [reply system-variables.md:1637]: 0 表示 又系统自动适配 max_threads 值, 1 表示并发度为 1

File: dynamic-config.md (hunk 2 above)
Comment (mengxin9014) [reply dynamic-config.md:317]: done

File: system-variables.md (hunk 4 above)
Comment (mengxin9014) [reply system-variables.md]: done

File: dynamic-config.md [hunk 5]
```diff
@@ -316,3 +312,42 @@ select @@tidb_slow_log_threshold;
...
+在线修改 TiFlash 配置，你也可以通过修改[系统变量](/system-variables.md)来实现。
+
+下面例子展示了如何通过变量 `tidb_max_tiflash_threads` 在线修改配置项 `max_threads`。
```
[the remark this answers is not in the document]
Comment (mengxin9014) [reply dynamic-config.md]: done

File: dynamic-config.md [hunk 6]
```diff
@@ -316,3 +312,42 @@ select @@tidb_slow_log_threshold;
...
+
+`max_threads` 默认值是 -1，表示此系统变量无效，由 TiFlash 的配置文件决定 max_threads，可以通过设置系统变量 `tidb_max_tiflash_threads` 将其修改为 10：
```
[the remark this answers is not in the document]
Comment (mengxin9014) [reply dynamic-config.md]: done

File: system-variables.md (hunk 3 above)
Comment (qiancai) [reply system-variables.md:1637]: 谢谢，可以在文档这里也加上 0 的含义吗？

Comment (mengxin9014) [reply system-variables.md]: done

**Review by qiancai — APPROVED** [review]

**Review by shichun-0415 — APPROVED** [review]

**Pull request merged**